dev-packages/browser-integration-tests/suites/tracing/metrics/pageload-resource-spans/test.ts

```diff
@@ -41,11 +62,84 @@ sentryTest('should add resource spans to pageload transaction', async ({ getLoca
   }
 
   expect(scriptSpans?.map(({ description }) => description).sort()).toEqual(expectedScripts);
+  expect(scriptSpans?.map(({ parent_span_id }) => parent_span_id)).toEqual(expectedScripts.map(() => spanId));
 
-  const spanId = eventData.contexts?.trace?.span_id;
+  const customScriptSpan = scriptSpans?.find(
+    ({ description }) => description === 'https://example.com/path/to/script.js',
+  );
 
-  expect(spanId).toBeDefined();
-  expect(imgSpans?.[0].parent_span_id).toBe(spanId);
-  expect(linkSpans?.[0].parent_span_id).toBe(spanId);
-  expect(scriptSpans?.map(({ parent_span_id }) => parent_span_id)).toEqual(expectedScripts.map(() => spanId));
+  expect(imgSpan).toEqual({
+    data: {
+      'http.decoded_response_content_length': expect.any(Number),
+      'http.response_content_length': expect.any(Number),
+      'http.response_delivery_type': '',
+      'http.response_transfer_size': expect.any(Number),
+      'network.protocol.name': 'unknown',
+      'network.protocol.version': 'unknown',
```

Comment:
I was hoping this would actually be populated but looks like PlayWright/Chromium Headless emits a `''` despite me returning a `Timing-Allow-Origin: *` header.

Reply:
Yeah, I'm also not super happy about this but it looks like some browser or playwright bugs might be responsible for the `''`. Coincidentally, we opened #15164 where it seems like at least some changes were made to cross origin performance entry data. So maybe this actually solves itself.